x/evm/keeper/params.go

```diff
@@ -51,7 +52,7 @@ func (k Keeper) GetLegacyParams(ctx sdk.Context) types.Params {
 
 // EnableDynamicPrecompiles appends the addresses of the given Precompiles to the list
 // of active dynamic precompiles.
-func (k Keeper) EnableDynamicPrecompiles(ctx sdk.Context, addresses ...string) error {
+func (k Keeper) EnableDynamicPrecompiles(ctx sdk.Context, addresses ...common.Address) error {
```

Comment:
I think this was string to avoid multiple conversions, can you perform a benchmark to see how much this affects the performance?

Reply:
I ran the benchmarks added by @facs95 on #2430 and the results show no degradation 👍

Reply:
Correct! Ran heavy benchmarks and there was no difference so we should be good with this changes